netbox_agent/network.py

```diff
@@ -60,7 +60,11 @@ def scan(self):
                 continue
 
             ip_addr = netifaces.ifaddresses(interface).get(netifaces.AF_INET)
-            ip_addr += netifaces.ifaddresses(interface).get(netifaces.AF_INET6)
```

Comment:
or we could go:
```suggestion
            ip_addr += netifaces.ifaddresses(interface).get(netifaces.AF_INET6, [])
```

Reply:
```suggestion
-            ip_addr = netifaces.ifaddresses(interface).get(netifaces.AF_INET)
+            ip_addr = netifaces.ifaddresses(interface).get(netifaces.AF_INET, [])
+            ip_addr += netifaces.ifaddresses(interface).get(netifaces.AF_INET6, [])
```

works like a charm on a VM without any IPs, not your case?